src/Components/GrowthChart/ChartSettingsButton/ChartSettingsButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { usePopper } from 'react-popper';
+import { EllipsisButton } from './EllipsisButton';
+import { PopoverList, PopoverListItem } from './PopoverList';
+
+type ChartSettingsButtonProps = {
+    setShowAnnotation: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+export const ChartSettingsButton = ({ setShowAnnotation }: ChartSettingsButtonProps) => {
+    const [referenceElement, setReferenceElement] = useState(null);
+    const [popperElement, setPopperElement] = useState(null);
+
+    const [isVisible, setIsVisible] = useState(false);
+
+    const { styles, attributes } = usePopper(referenceElement, popperElement, {
+        placement: 'bottom',
+        modifiers: [
+            {
+                name: 'offset',
+                options: { offset: [0, 5] },
+            },
+        ],
+    });
+
+    return (
+        <div className='absolute top-2 right-2'>
```

Comment:
this is just to show how the styling will be like, ofc we wont have it "into" the graph, but i mean that this should be a part of the component